Makefile

```diff
@@ -2,20 +2,42 @@
 # Use of this source code is governed by a Zero-Clause BSD license that can
 # be found in the tests/TESTS_LICENSE file.
```

Comment:
I think it's a bit strange that this file is governed by a license file from a subdirectory.

Reply:
done.